source/common/runtime/runtime_features.cc

```diff
@@ -93,6 +93,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_thrift_connection_draining);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_http2_use_oghttp2);
 // Used to track if runtime is initialized.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_runtime_initialized);
+// TODO(mattklein123): Flip this to true and/or remove completely once verified by Envoy Mobile.
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_android_always_use_v6);
```

Comment:
Also please unit test, or add unit tests to the TODO here if the timing is too tight.

Reply:
Do you have a suggestion of how to unit test this in this repo? This is all behind an android only ifdef.

Reply:
I think I would prefer to not have a build flag for it since that will be harder to test. I think Alyssa's suggestion is good to just refactor it and add a unit test. I can do that but will do it as a follow up.